### PR TITLE
GRASS GIS 7.8.4 released

### PR DESCRIPTION
**Pull request by neteler**

### CODE REVIEW

File: content/news/2020_10_05_GRASS_GIS_7_8_4R_released.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+**Binaries/Installer download:**\
+
+- winGRASS 7.8.4/standalone: 
+  [32bit standalone installer](/grass78/binary/mswindows/native/x86/WinGRASS-7.8.4-1-Setup-x86.exe) \| [64bit standalone installer](/grass78/binary/mswindows/native/x86_64/WinGRASS-7.8.4-1-Setup-x86_64.exe)
+- winGRASS 7.8.4/OSGeo4W:
+  [32bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/osgeo4w-setup-x86.exe) \| [64bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/osgeo4w-setup-x86_64.exe)
+- [Debian](https://tracker.debian.org/pkg/grass)
+- [Fedora/Centos/EPEL](https://src.fedoraproject.org/rpms/grass)
+- [Ubuntu](https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-experimental/+packages?field.name_filter=grass)
```
Comment (landam) [inline content/news/2020_10_05_GRASS_GIS_7_8_4R_released.md:34]: Today grass 7.8.4 package will be moved to Ubuntu Unstable.

Comment (neteler) [reply content/news/2020_10_05_GRASS_GIS_7_8_4R_released.md:34]: Seems to be there now:
https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-unstable/

Comment (neteler) [inline content/news/2020_10_05_GRASS_GIS_7_8_4R_released.md:34]:
```suggestion
- [Ubuntu](https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-unstable/+packages?field.name_filter=grass)
```

File: content/news/2020_10_05_GRASS_GIS_7_8_4R_released.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+catalogue. Many display modules received fixes as well, and the vector
+digitizer now works as expected. 
+
+The overview of new features in the 7.8 release series is available at 
+[new features in GRASS GIS 7.8](https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures78).
+See also our detailed announcement with the full list of changes and 
+bugs fixed at 
+[https://trac.osgeo.org/grass/wiki/Release/7.8.4-News](https://trac.osgeo.org/grass/wiki/Release/7.8.4-News).
+
+**Binaries/Installer download:**\
```
Comment (neteler) [inline content/news/2020_10_05_GRASS_GIS_7_8_4R_released.md:26]:
```suggestion
**Binaries/Installer download:**
```

**Pull request merged**